<a href="https://colab.research.google.com/github/Avril009d/Avril009d/blob/main/Beginner_Code_Example.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **Beginner Code Example**

*We are going to create a simple user-defined function that calculates the total for an income increase*

Create a list of what we want to calculate
- Gross amount
- ~Net Amount~
- Percentange of increase

In [ ]:
#Create a user defined function
#Unlike programming languages such as C++ or JAVA, Python is dynamically typed. This means that the interpreter determines the type of a variable at runtime based on the value assigned to it.

def salary_increase(gross_sal:float,percentage:float):
  your_name = input('Enter in your first name: ')
  yn = your_name.upper()
  increase = gross_sal * percentage
  new_salary = round(gross_sal + increase,2)
  return f'{yn} your new salary is: ${new_salary:,}'


### **Call the function in three ways**

In [ ]:
#Ask for user input for what values to pass as arguments
current_sal = float(input('Enter in your current salary: '))
per_increase = float(input('Enter in the percentage that your salary is increasing [ex:.03 for 3%]: '))

#Call the function and then pass in the inputs as agruments
salary_increase(current_sal,per_increase)

Enter in your current salary: 75000.75
Enter in the percentage that your salary is increasing [ex:.03 for 3%]: .05
Enter in your first name: olivia


'OLIVIA your new salary is: $78,750.79'

In [ ]:
#Call the function and manually add the arguments
salary_increase(gross_sal=85000,percentage=.03)

Enter in your first name: olivia


'OLIVIA your new salary is: $87,550.0'

In [ ]:
#Call the function and manually add the arguments
salary_increase(85000,.04)

Enter in your first name: Jason


'JASON your new salary is: $88,400.0'

**Salary Table**

|Salary| Year|
|------|-----|
|Amount|Year1|
|Amount2|Year2|
|Amount3|Year3|


## -----------------------------------------------------

*Fill in the table below with your salary inputs above*

Double click this cell and edit the table by highlighting and replacing "Amount" with one of the numbers above and when finished.

|Salary| Year|
|------|-----|
|78,751|2024|
|87,550|2025|
|88,400|2026|


# Resource for Python Learning:
Select the link......

[W3 Schools](https://www.w3schools.com/python/python_intro.asp)